op-challenger/game/scheduler/coordinator.go

```diff
@@ -49,18 +52,31 @@ func (c *coordinator) schedule(ctx context.Context, games []common.Address) erro
 		}
 	}
 
+	var gamesInProgress int
+	var gamesChallengerWon int
+	var gamesDefenderWon int
 	var errs []error
+	var jobs []job
 	// Next collect all the jobs to schedule and ensure all games are recorded in the states map.
 	// Otherwise, results may start being processed before all games are recorded, resulting in existing
 	// data directories potentially being deleted for games that are required.
-	var jobs []job
 	for _, addr := range games {
 		if j, err := c.createJob(addr); err != nil {
 			errs = append(errs, err)
 		} else if j != nil {
 			jobs = append(jobs, *j)
+
+			switch j.status {
+			case types.GameStatusInProgress:
+				gamesInProgress++
+			case types.GameStatusDefenderWon:
+				gamesDefenderWon++
+			case types.GameStatusChallengerWon:
+				gamesChallengerWon++
+			}
```

Comment:
This is the right idea, but we should also count games that we didn't create a job for. Otherwise we would miss counting games that are already in flight and so we're not scheduling a second job for. 

Either `createJob` will need to return the game status or once `createJob` has run its safe to assume the `gameState` exists in the tracking map so you could just grab the status from there - not sure which is cleaner/safer...

Reply:
Attempted to fix this - read the state on the non-matching case with an ok check.